front_end/webserver.py

```diff
@@ -1,3 +1,71 @@
+from HomeHandler import *
```

Comment:
You should be able to force it to load everything in the handlers directory, rather than listing these out individually. [This page](https://stackoverflow.com/questions/32239372/how-to-import-files-in-python-using-sys-path-append) should explain how to do it. You would need to add `/app/handlers` as a path, I believe.